website/content/partials/known-issues/ephemeral-loggers-memory-leak.mdx

```diff
@@ -1,14 +1,39 @@
 ### Vault is storing references to ephemeral sub-loggers leading to a memory leak
 
-Vault is unexpectedly storing references to ephemeral sub-loggers which prevents them from being cleaned up, leading to
-a memory leak. This impacts many areas of Vault, but primarily logins in Enterprise.
+In Vault versions predating 1.13.7, 1.14.3, and 1.15.0 Vault does not honor a
+modified `log_level` configuration for certain subsystem loggers on SIGHUP.
+
+This is known to affect the `resolver.watcher` and `replication.index.*`
+sub-loggers.
+
+This can be seen after modifying the `log_level` and issuing a reload (SIGHUP),
+For example, after starting a server with `log_level: "trace"` and modifying it
+to `log_level: "info"` the following lines appear after reload:
+
+```
+[TRACE] resolver.watcher: dr mode doesn't have failover support, returning
+...
+[DEBUG] replication.index.perf: saved checkpoint: num_dirty=5
+[DEBUG] replication.index.local: saved checkpoint: num_dirty=0
+[DEBUG] replication.index.periodic: starting WAL GC: from=2531280 to=2531280 last=2531536
+```
+
+A change was introduced to resolve this issue:
+`core: All sub-loggers now reflect configured log level on reload. [GH-22038]`
+
+As a result of this change Vault would unexpectedly store references to ephemeral sub-loggers which prevents them from
+being cleaned up, leading to a memory leak. This impacts many areas of Vault, but primarily logins in Enterprise.
 There is no workaround.
 
+The change that introduced this leak will be reverted in the next release, restoring the previously known behavior of
+sub-loggers not updating log levels on SIGHUP.
+
 #### Affected versions
 
 This issue affects Vault Community and Enterprise versions:
```

Comment:
Since we mention multiple issues above, should we call out which specific issue we're referring to here?
```suggestion
The memory leak issue affects Vault Community and Enterprise versions:
```